docs/pivotal_tracker.md

```diff
@@ -33,7 +33,7 @@ Next create the integration in Aha!
 3. Choose the integration in Pivotal Tracker that you created in step 2 above.
 4.	Create a new _Activity Web Hook_ for v5 in Pivotal Tracker and paste in the Hook URL.
 5. Enable the integration.
-6. Test the integration by going to one of your features in Aha! and using the _Send to Pivotal Tracker_ item in the _Actions_ menu on the features page. You should then look at your project in Pivotal Tracker and see that the feature (and any requirements) were properly copied. 
+6. Test the integration by going to one of your features in Aha! and using the _Send to Pivotal Tracker_ item in the _Send_ dropdown next to the Integrations field on the features page. You should then look at your project in Pivotal Tracker and see that the feature (and any requirements) were properly copied.
```

Comment:
Should "Integrations field" be capitalized? It's in lower case in other places (rally.md)